pkg/tinydtls/Makefile

```diff
@@ -1,7 +1,9 @@
 PKG_NAME=tinydtls
 PKG_URL=https://github.com/rfuentess/TinyDTLS.git
-# PKG_VERSION=RIOT-OS
-PKG_VERSION=d06ad84d3e8bd3e86f7557faee34b68d5f8c0129
+PKG_VERSION=master
```

Comment:
Do you want to actually use the HEAD of the master branch?

We use fixed commits or tags to ensure proper rebuilds.